docs/Configuration.md

```diff
@@ -1,5 +1,42 @@
 # Configuration
 
+## Table of contents
+
+* [Introduction](#Introduction)
+* [DB section](#DB-section)
+  * [engine](#engine)
+  * [user](#user)
+  * [password](#password)
+  * [database_host](#database_host)
+  * [database_name](#database_name)
+  * [polling_interval](#polling_interval)
+* [MYSQL section](#MYSQL-section)
+  * [host](#host)
+  * [port](#port)
+  * [user](#user)
+  * [password](#password)
```

Comment:
missing suffix to avoid conflicts with the previous headings:
```
  * [user](#user-1)
  * [password](#password-1)
```

Reply:
Good that you checked and found this.